src/cmd-build

```diff
@@ -50,6 +51,10 @@ while true; do
             shift
             VERSION=$1
             ;;
+        --parent)
```

Comment:
Would be nicer if we had https://github.com/coreos/coreos-assembler/issues/531 - this would just be a `parent: ...` in the buildconfig or so.